src/app/migrate.py

```diff
@@ -32,8 +32,18 @@
                               "                    comment TEXT, \n"
                               "                    CONSTRAINT passwords_pk PRIMARY KEY (pass_id) \n"
                               "                    ) WITH OIDS;")
+SQL_create_table_session_objects = ("CREATE TABLE session_objects ( \n"
```

Comment:
Why are you using raw SQL?

Reply:
This part(migrate.py) will be refactored by Vitaly. I use raw SQL only as temporary solution.

Reply:
Ok